package.json

```diff
@@ -50,7 +50,7 @@
     "@emotion/react": "^11.13.3",
     "@emotion/styled": "^11.13.0",
     "@formatjs/intl-listformat": "^7.7.10",
-    "@mui/material": "^6.4.5",
+    "@mui/material": "^5.15.10",
```

Comment:
Suspect this needs to stay in sync with https://github.com/theopensystemslab/planx-new/blob/main/editor.planx.uk/package.json#L15 else we can get local PNPM issues